baselines/webworker.generated.d.ts

```diff
@@ -2049,6 +2010,26 @@ declare var XMLHttpRequestUpload: {
     new(): XMLHttpRequestUpload;
 };
 
+declare namespace console {
```

Comment:
this is going to be a breaking change. i will need to run the tests against definitelytyped.

Reply:
Would `type Console = typeof console;` mitigate the compat problem?

Reply:
My concern is that augmentation to `Console` are now ignored. making it a type makes it an error. but does not address the issue.

Reply:
My quick search on DefinitelyTyped (with a search query `interface Console `) says only [arangodb](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/arangodb/index.d.ts) augments the existing `Console` definition, other types including Node.js and react-native have their own ones.

Reply:
`extends Console` also only shows core-js.

Reply:
The `arangodb` type is not really about augmenting existing browser console but about typing their own console module: https://github.com/arangodb/arangodb/blob/16a1fabf2057fb1da56a7abd44bd8e27ab1bead5/js/common/bootstrap/modules/console.js

`core-js` dropped their `core.log` that was using `extends Console` 4 years ago. (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36100)